iceberg-theme/static/css/iceberg-theme.css

```diff
@@ -370,10 +370,6 @@ i.fa.fa-chevron-down {
     }
   }
 
-@media screen and (max-width: 1280px) {

```

Comment:
Instead of removing this completely, should we maybe decrease the width? At 768px the hamburger appears in the top-left that allows opening/closing the side menu.